### ISSUE TITLE
Using symbols as keys for object properties

### ISSUE THREAD

**Issue by niwsa** [HerringtonDarkholme/typescript-repl#9]
I am not getting the expected output in tsun as per docs [http://www.typescriptlang.org/docs/handbook/symbols.html](url) 

```
> let sym = Symbol();
undefined
> sym
Symbol()
> let obj = {
..[sym]: "value"
..};
undefined
> obj[sym]
undefined
> console.log(obj[sym])
undefined
> 
```

Version info:

```
$ sudo npm list -g tsun
/usr/local/lib
`-- tsun@0.2.2 
```

```
$ sudo npm list -g typescript
/usr/local/lib
+-- tsun@0.2.2
| `-- typescript@1.8.10 
+-- typescript@1.8.10 
`-- typings@1.0.4
  `-- typings-core@1.0.1
    `-- typescript@1.8.7 
```

**Comment by HerringtonDarkholme** [comment]
This is a more general bug that computed property is wrongly translated.

**Comment by HerringtonDarkholme** [comment]
Fixed in 1f24dbcb42663ee3d34e8e59bf8b56e5712721be
